modules/basis/src/lib/parsers/parse-pvr.js

```diff
@@ -3,8 +3,9 @@
 // Copyright (c) 2017 Tarek Sherif, The MIT License (MIT)
 import {GL} from '../gl-constants';
 
-const PVR_CONSTANTS = {
+export const PVR_CONSTANTS = {
   MAGIC_NUMBER: 0x03525650,
+  MAGIC_NUMBER_EXTRA: 0x50565203,
```

Comment:
Can we have better names? How are these different? Any links?

Reply:
For DDS this byte is called "VERSION" there are two possible versions. This information is described in spec: 
http://cdn.imgtec.com/sdk-documentation/PVR+File+Format.Specification.pdf